src/usr/local/pkg/miniupnpd.xml

```diff
@@ -72,7 +71,7 @@
 			<sethelp>
 				<![CDATA[
 				Enables support for client requests using Port Control Protocol (PCP) and its
-				compatible predecessor NAT Port Mapping Protocol (NAT-PMP).
+				predecessor NAT Port Mapping Protocol (NAT-PMP).
```

Comment:
The current text is fine given that, AFAIK, PCP supports NAT-PMP as well.

Reply:
> A PCP-only gateway receiving a NAT-PMP request (identified by the first octet being zero) will interpret the request as a version mismatch. Normal PCP processing will emit a PCP response that is compatible with NAT-PMP, without any special handling by the PCP server.

https://datatracker.ietf.org/doc/rfc6887/

It sounds to me like leaving "compatible" in the pfSense GUI is fine.

Reply:
Right. I interpret the sentence that follows as stating what "normally" happens in the case of a version mismatch.

Reply:
Unless it's shown that pfSense/miniupnpd won't handle NAT-PMP clients when only PCP is enabled, let's keep the "compatible" part.